src/App.tsx

```diff
@@ -7,7 +7,7 @@ import { loadFiles } from './FileLoader'
 import { Roster as Roster2018 } from './types/KillTeam2018'
 import { Roster as Roster2021 } from './types/KillTeam2021'
 
-export function App () {
+export function App (): JSX.Element {
```

Comment:
I, personally, don't like to explicitly state the output type. TS can calculate that well on its own.

Reply:
Alright, I see that this is required by the standard linter; D. All good, when you merge the present PRs, I can open a PR with the remaining fixes.

Reply:
Btw, if we have to do it, I'd go for importing `FC` from react and exposing that. In `App` that doesn't change much, but in others we can expose `FC<Props>`, and it will typesafe proper props passed in component rendering. Just setting it as `JSX.Element` is erasing props data.

Reply:
Totally up for a better way of doing this! I'm feeling my way round Typescript and React with this project, so all recommendations greatly appreciated. I'll look into `FC` :+1:

Reply:
Ah, brilliant, that makes perfect sense.